src/app/backend/handler/localehandler.go

```diff
@@ -131,46 +122,40 @@ func (handler *LocaleHandler) determineLocalizedDir(locale string) string {
 	locales := handler.SupportedLocales
 	tag, _, confidence := language.NewMatcher(locales).Match(tags...)
 
-	if confidence < language.Exact {
-		tag, confidence, err = mapLocale(locale, locales)
-		if err != nil {
-			return defaultDir
-		}
-	}
-
-	matchedLocale := tag.String()
-	// If locale match is exact, then we have to manually look for proper locale code as language
-	// library contains a bug that returns invalid locale string.
-	// Related issue: https://github.com/golang/go/issues/24211
-	if confidence == language.Exact {
-		matchedLocale = ""
-		for _, l := range locales {
-			base, _ := tag.Base()
-			if l.String() == base.String() {
-				matchedLocale = l.String()
+	localeDir := ""
```

Comment:
Why did you change the original logic here and removed string replacer? Previous version should work just fine. 

1. Replace 
2. Try to match the language
3. If an exact match, look for proper locale dir (can be omitted once go/text bug is fixed)
4. Check if dir exists and return

Reply:
I found the other solution that fix our issue without removing temporary fix for the bug and replacing string.
According to the spec of `language` module, we can use not only `Base` of the language but also `Parent`.
Using `Parent`, we can map all languages that we support now, without string replaces.
More details, see https://github.com/kubernetes/dashboard/issues/4973

Also, I added comments on my codes.

Reply:
I updated my investigation #4973 after update Angular to 9.0.5, implementations are getting better. 😉